testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/admin/realm/RealmTest.java

```diff
@@ -692,6 +692,15 @@ public void updateRealm() {
         assertEquals(Boolean.FALSE, rep.isRegistrationEmailAsUsername());
         assertEquals(Boolean.FALSE, rep.isEditUsernameAllowed());
         assertEquals(Boolean.FALSE, rep.isUserManagedAccessAllowed());
+
+        rep.setAccessCodeLifespanLogin(0);
+        rep.setAccessCodeLifespanUserAction(0);
+        try {
+            realm.update(rep);
```

Comment:
Can you please add something like `Assert.fail("Not expected to successfully update the realm");` after the `realm.update(rep)` ?

If the line with "Assert.fail" is not added, then it is not tested if realm update really failed as expected due the incorrect validation. See some other places for the inspiration (For example method `deleteSession()` of the `RealmTest`).